dev-tools/openapi-manager/src/output.rs

```diff
@@ -202,31 +217,48 @@ pub(crate) mod headers {
     pub(crate) static GENERATING: &str = "Generating";
 
     pub(crate) static UP_TO_DATE: &str = "Up-to-date";
-    pub(crate) static STALE: &str = "Stale";
-    pub(crate) static MISSING: &str = "Missing";
+    pub(crate) static OUT_OF_DATE: &str = "Out-of-date";
+    pub(crate) static STALE: &str = "-> Stale";
```

Comment:
For my own education if/when I encounter errors from this tool: what's the difference between "out of date" and "stale"?

Reply:
Actually, good question. I think I'm going to change this to:

* Fresh
* Stale
  -> Modified
  -> New